hera_pspec/grouping.py

```diff
@@ -153,6 +154,15 @@ def average_spectra(uvp_in, blpair_groups=None, time_avg=False,
         does this for every specified key. Every stats_array key that is
         not specified is thrown out of the new averaged object.
 
+    if_error_weighted: bool, optional
+        If True, power spectra will be averaged in a way weighed by
+        error bars. Default: False
+
+    name_error_weights: string, optional
```

Comment:
Instead of having two arguments needed to specify the weights, perhaps we could only use one? e.g. one kwarg called `error_weights` that is `None` by default (the same as setting `if_error_weighted = False`), but can be set to a string that tells the function which key of `stats_array` to use.

Reply:
Sure.